index.html

```diff
@@ -29,6 +29,32 @@ <h3 class="contacts__heading">Пишите:</h3>
+          <ul class="footer__social-media">
+            <li class="footer__item">
+              <a class="footer__link" href="#">
+                <svg class="footer__social-icon" xmlns="http://www.w3.org/2000/svg" width="18" height="14" fill="none"><path fill="#D22856" d="M11.564.169A3.675 3.675 0 0 1 13.672.14c.688.196 1.291.59 1.723 1.122.71-.21 1.41-.447 2.131-.656a2.03 2.03 0 0 1-.253.917c-.157.284-.38.533-.653.729.253.026.508-.002.747-.081.24-.08.456-.208.633-.376-.099.414-.3.801-.588 1.131-.287.33-.653.594-1.07.772-.247 6.27-3.367 10.467-10.689 10.296H5.15c-.443 0-4.418-.428-5.149-1.75 1.752.245 3.539-.147 4.984-1.095-1.03-.286-2.852-.447-3.161-2.74.388.133.81.163 1.215.085C1.76 7.705.34 7.039.422 4.993c.398.306.905.462 1.421.438C1.102 5.203-.247 2.31.896.825c1.926 1.723 3.964 3.35 7.59 3.512C8.711 2.205 9.69.777 11.563.169Z"/></svg>
+              </a>
+            </li>
+            <li class="footer__item">
+              <a class="footer__link" href="#">
+                <svg class="footer__social-icon" xmlns="http://www.w3.org/2000/svg" width="9" height="18" fill="none"><path fill="#D22856" d="M6.509 3.193H9V0H6.02C2.762.171 1.6 2.271 2.077 5.816H0v3.336h2.076V18h3.945V9.152h2.958V5.816H6.01c-.02-.998-.041-2.29.499-2.623Z"/></svg>
+              </a>
+            </li>
+            <li class="footer__item">
+              <a class="footer__link" href="#">
+                <svg class="footer__social-icon" xmlns="http://www.w3.org/2000/svg" width="15" height="13" fill="none"><path fill="#D22856" d="M1.834 12.809a1.257 1.257 0 0 1-1.285.03 1.097 1.097 0 0 1-.443-.457.97.97 0 0 1-.092-.603V1.221A.97.97 0 0 1 .106.618C.2.429.355.27.549.16a1.256 1.256 0 0 1 1.285.031l12.4 5.502c1.022.445 1.022 1.169 0 1.614l-12.4 5.502Z"/></svg>
+              </a>
+            </li>
+          </ul>
+          <div class="footer__copyright">
+            <p class="footer__author">Разработано</p>
+              <svg xmlns="http://www.w3.org/2000/svg" width="27" height="34" fill="none"><path fill="#D22856" d="M13.643.016 13.5 0 0 1.407v24.606L13.5 34 27 26.013V1.407L13.643.016Zm11.591 25L13.5 31.956 1.766 25.015V14.93l11.685 6.916.02 1.253-8.017-4.743-.011 1.225 8.046 4.809.02 1.286-8.045-4.758-.01 1.226L13.5 26.95l8.124-4.833V16.91l3.62-2.15-.01 10.256Zm0-11.564-3.22 1.88-1.482.875-7.065-4.187-.011 1.225 6.04 3.573-.053.031-.13.076-.86.492-4.986-2.948-.011 1.225 3.936 2.328-.933.61-.021.012-2.971-1.742-.011 1.225 1.947 1.135-1.988 1.186-11.618-6.872 11.638-6.958 11.8 6.834Zm0-1.225L13.44 5.31 1.766 12.22V2.99L13.5 1.764 25.234 2.99v9.237Z"/></svg>
```

Comment:
@ElenaPodv The Academy's logo is, in fact, a link as well 😀 Please wrap it in a `<a>` tag, add a visually hidden `<span>`, reassign with suitable class names, and include the attributes `aria-hidden="true"` and `focusable="false"` on the SVG. Additionally, style the `:hover `and `:active` states as those are defined in the [Styleguide](https://www.figma.com/file/TYlLseXoMXDflJYbU1DGHa/HTML-2-%2F-%D0%9F%D0%B8%D0%BD%D0%BA?type=design&node-id=1226%3A1046&mode=design&t=aWx0M6s4oKboJZOE-1).

Reply:
@naumch1k Academy Logo Link:
Wrapped the Academy's logo in an <a> tag, making it a clickable link for improved user navigation.
Visually Hidden Span: Introduced a visually hidden <span> containing meaningful text for the Academy's logo link to enhance accessibility for users with visual impairments.
Class Reassignment:Reassigned suitable class names to the logo link and accompanying elements to maintain consistency and clarity in our codebase.
Aria Attributes for SVG: Added aria-hidden="true" and focusable="false" attributes to the SVG element within the logo link. This signifies that the SVG is decorative and should be ignored by screen readers, further enhancing accessibility.
Styled :hover and :active States: Applied styles to the :hover and :active states of the logo link in accordance with the guidelines outlined in our Styleguide. This ensures a consistent and polished visual experience for users interacting with the logo.